### DISCUSSION TITLE
Redirect not work, but display favicon

### DISCUSSION THREAD

**Discussion by TK88777** [YOURLS/YOURLS#3416] [Q&A]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.9.1

### Description

Redirect not work, but display favicon of Yourls :-/
- Apache/**2.4** 
- PHP **7.4**  

### Expectation

Redirection is not made, but  i get favicon...

mydomain.tld/yourls -- > https://yourls.org/
mydomain.tld/yourlsblog --> https://blog.yourls.org/

### Reproduction steps

1. Install fresh and last Yourls 1.9.1
2. Install language (french)
3. Tests URLs already present after install  (yourls, yourlsblog)
4. Redirected to favicon...

### Context

Running :
-  **Apache/2.4**.38 (Debian) 
- **PHP 7.4**.29
- ReverseProxy (SSL)

**Comment by dgw** [comment]
Issues with shortlinks not redirecting as expected are almost always because of a mistake in the server configuration.

Please double check https://docs.yourls.org/guide/server-configuration.html#apache

**Comment by TK88777** [comment]
Hello,

I have checked the configuration of my server several times and the htaccess is well created (after install).
Also log in each request for analysis.
I even tried to get direct access (without my ReverseProxy).
And I even set up a few additional codes to log in each call to the file " yourls-loader.php " 
(....)

This problem does not seem to affect all URLs, only some.
